src/v2/Apps/Order/Components/PriceOptions.tsx

```diff
@@ -143,19 +149,31 @@ export const PriceOptions: React.FC<PriceOptionsProps> = ({
               trackClick("Different amount", 0)
             }}
             key="price-option-custom"
+            name="radio"
```

Comment:
Is a `name` needed here? I couldn't find where it's been used. If it's needed, would it make sense to use a more descriptive name?

Reply:
I forgot if I added these for it to work well with RTL, but I'll take a look.

Reply:
I saw RTL is [getting them by role](https://github.com/artsy/force/pull/9118/files#diff-e3c631e6b08c80b65bdfee8975885955c11cf9850f8fb85bfc7e38657b1e10a9R154).

Reply:
Yeah, I must've left it there by mistake. Removing.